apps/mull-api/src/app/event/event.service.ts

```diff
@@ -51,7 +51,7 @@ export class EventService {
    * Find all the events that the user has not joined, created or is not co-hosting
    * @param userId
    */
-  findDiscoverEvent(userId: number): Promise<Event[]> {
+  discoverEvents(userId: number): Promise<Event[]> {
```

Comment:
How do we feel about this
```suggestion
  recommendedEvents(userId: number): Promise<Event[]> {
```

Reply:
That or `getRelevantEvents`, what do you think?

Reply:
Oh, or `getEventsRecommendedToUser`, to make it match up better with the other methods

Reply:
Yeah, that works if it better matches the other methods